osprey-mock-service.js

```diff
@@ -48,6 +48,14 @@ function createServerFromBaseUri (raml, options) {
   var app = osprey.Router()
   var path = (raml.baseUri || '').replace(/^(\w+:)?\/\/[^/]+/, '') || '/'
 
+  if (options.definition) {
+    app.use('/resources', function (req, res) {
+      var body = '< link:"' +
```

Comment:
I think the `< ` is extraneous. It appears in the example of #33 because that's how `curl` formats headers returned in responses.

Also, to be clear, this should be returned as a `header`, not in the `body`.